src/DataValidation.py

```diff
@@ -259,6 +259,23 @@ def checkStartingItemsForValidItemsAndCategories():
                     if len([item for item in DataValidation.item_table if "category" in item and category_name in item["category"]]) == 0:
                         raise ValidationError("Item category %s is set as a starting item category, but is misspelled or is not defined on any items." % (category_name))
 
+    @staticmethod
+    def checkStartingItemsForBadSyntax():
+        if "starting_items" not in DataValidation.game_table:
+            return
+        
+        starting_items = DataValidation.game_table["starting_items"]

```

Comment:
These can be combined into one line. 
`if not starting_items := DataValidation.game_table.get("starting_items", False):
  return`

Don't know that it's more readable but it is more "pythonic" with the addition of the `:=` operator

Reply:
Good point! I actually completely forgot that the walrus operator was a thing. 😜